README.md

```diff
@@ -1,19 +1,33 @@
 # SnarkyJS &nbsp; [![npm version](https://img.shields.io/npm/v/snarkyjs.svg?style=flat)](https://www.npmjs.com/package/snarkyjs) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/o1-labs/snarkyjs/blob/main/CONTRIBUTING.md)
 
-The easiest way to write zk programs is using SnarkyJS, a TypeScript framework for zk-SNARKs and zkApps.
+The SnarkyJS SDK helps developers build apps powered by zero-knowledge cryptography.
 
-To write a zkApp smart contract for the Mina Protocol, we recommend using the [zkApp CLI](https://github.com/o1-labs/zkapp-cli). The zkApp CLI allows you to scaffold, write, test, and deploy zkApps using recommended best practices. zkApps created with the zkApp CLI include SnarkyJS and provide project scaffolding, a test framework, and correct formatting.
+SnarkyJS is a TypeScript library for writing smart contracts based on zero-knowledge proofs for the Mina Protocol. The easiest way to write zero-knowledge (zk) programs is using the SnarkyJS TypeScript framework for zk-SNARKs and zkApps. 
 
-To learn more, see the [SnarkyJS reference](https://docs.minaprotocol.com/en/zkapps/snarkyjs-reference) and [zkApp Developers](https://docs.minaprotocol.com/zkapps) documentation.
+SnarkyJS is automatically included when you write a zkApp smart contract for the Mina Protocol using the [Mina zkApp CLI](https://github.com/o1-labs/zkapp-cli). 
 
-For a list of changes between versions, see the [CHANGELOG.md](https://github.com/o1-labs/snarkyjs/blob/main/CHANGELOG.md).
```

Comment:
I think we should keep this sentence and also one about the docs. Removing context about what this is and dumping them in a "quick links" section makes the CHANGELOG / docs links less visible, which isn't to our advantage.

I'd advocate for rolling back the introduction of "quick links" and instead write more text about all of these links.